<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labdlina">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Длина стороны:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labug1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Угол 1(градусы):</string>
    </property>
   </widget>
   <widget class="QLabel" name="labug2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Угол 2(градусы):</string>
    </property>
   </widget>
   <widget class="QLabel" name="eror">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>320</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Ошибка</string>
    </property>
   </widget>
   <widget class="QLabel" name="labpl1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Площадь:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labpr1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Периметр:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lest">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leug">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="leug2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labpl2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>190</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labpr2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbvich">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычислить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
